packages/experiments-realm/crm/contact.gts

```diff
@@ -258,18 +272,22 @@ export class PhoneField extends FieldDef {
   };
 }
 
-class FittedTemplate extends Component<typeof Contact> {
+class EmbeddedTemplate extends Component<typeof Contact> {
   <template>
-    <article class='fitted-contact-card'>
+    <article class='embedded-contact-card'>
       <div class='avatar-container'>
```

Comment:
Let's make this a component in `experiments-realm/components` from this because it is used in multiple places.